docs/JestObjectAPI.md

```diff
@@ -115,10 +126,20 @@ test('original implementation', () => {
 });
 ```
 
+
+:::
+
+:::note
+
+Note: this method was previously called `autoMockOn`. When using `babel-jest`, calls to `enableAutomock` will automatically be hoisted to the top of the code block. Use `autoMockOn` if you want to explicitly avoid this behavior.
```

Comment:
Could you look through this file again? Seems like you are only adding new lines, which are duplicating information which already exists in the document.

Reply:
I removed the new lines and duplicated messages.  Please let me know if there's anything else needed to change.